ticker/src/main/java/com/robinhood/ticker/TickerColumn.java

```diff
@@ -62,10 +61,10 @@
     private float previousBottomDelta;
     private int directionAdjustment;
 
-    TickerColumn(char[] characterList, Map<Character, Integer> characterIndicesMap,
+    TickerColumn(List<char[]> characterLists, List<Map<Character, Integer>> characterIndicesMaps,
```

Comment:
Just looking at List<Map<Character, Integer>> makes me wonder if we should write a simple abstraction on top of it. It's not awful, but it's not simple either. nbd either way.